spec/support/matchers.rb

```diff
@@ -1,3 +1,6 @@
+require 'rspec/expectations'
```

Comment:
That's a good point -- if users `require 'rspec/matchers/fail_matchers` and then try `RSpec::Matchers.define` they would run into the same issue.  So I agree with @JonRowe.